src/screens/InputScreen.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import {SafeAreaView, StyleSheet} from 'react-native';
+
+import NavigationHeader from '../components/NavigationHeader';
+import {Input, Icon, Text} from 'react-native-elements';
+
+class InputScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: 'Unknown',
```

Comment:
why is `Unknown`?

Reply:
Instead of creating additional view with size for text, I decided to keep text always present on the screen, because the text area without text collapses. 'Unknown' is first think came up in my mind to use as dummy :)

Reply:
shouldn't you move this value to the constant? since it's used twice